packages/dpkg.rb

```diff
@@ -0,0 +1,38 @@
+require 'package'
+
+class Dpkg < Package
+  description 'dpkg, a medium-level package manager for Debian'
```

Comment:
You can remove `dpkg,` from the description since `crew search dpkg` will generate `dpkg: dpkg, a medium-level...` which is a bit redundant.